src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewController.swift

```diff
@@ -59,15 +59,14 @@ class HealthCertificateViewController: UIViewController, UITableViewDataSource,
 		viewModel.markAsSeen()
 
 	}
-//		override func viewDidDisappear(_ animated: Bool) {
-//		super.viewDidDisappear(animated)
-//
-//		parent?.navigationController?.navigationBar.backgroundAlpha = 1.0
-//	}
 	
 	override func viewWillDisappear(_ animated: Bool) {
 	super.viewWillDisappear(animated)
 
+	if let dismissHandlingNC = navigationController as? DismissHandlingNavigationController {
+		dismissHandlingNC.restoreOriginalNavigationBar()
+	}
+		
 	parent?.navigationController?.navigationBar.backgroundAlpha = 1.0
```

Comment:
Please indent.

Reply:
Indentation is alright, it's GitHub issue, otherwise our SwiftLint will complain ;)

Reply:
believe me ;)

<img width="835" alt="HealthCertificateViewController" src="https://user-images.githubusercontent.com/22373291/193771596-781c5166-589b-4ece-a59e-09a2160509ee.png">

Reply:
Fixed it, thanks for the catch.